app/services/decisions/simple_decision_tree.rb

```diff
@@ -10,11 +10,12 @@ class SimpleDecisionTree < BaseDecisionTree
       claim_details: :work_item,
       work_item: :work_items,
       disbursement_cost: :disbursements,
-      other_info: :solicitor_declaration,
+      check_answers: :solicitor_declaration
```

Comment:
This doesn't have to use the simple decision tree... as there si no input it could just be a hardcoded link on the screen if that is easier.....